app/component/contribute/contribute_card_component.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Contribute
+  class ContributeCardComponent < ViewComponent::Base
```

Comment:
```suggestion
  class CardComponent < ViewComponent::Base
```

Considering the module name is `Contribute`, let's not prefix that for class!

Reply:
Do the same for file